lib/miq_v2v_ui/engine.rb

```diff
@@ -10,5 +10,12 @@ def vmdb_plugin?
       app.config.assets.paths << root.join('assets', 'images').to_s
     end
 
+    initializer 'plugin' do
+      Menu::CustomLoader.register(
+        Menu::Section.new(:migration, N_("Migration"), 'fa fa-plus', [
+          Menu::Item.new('overview', N_('Overview'), 'miq_report', {:feature => 'miq_report', :any => true}, '/migration')
```

Comment:
can you also add this one @ZitaNemeckova ?
https://github.com/priley86/manageiq-ui-classic/commit/f8792b86b97ee9e8133e700529d093f9845060fd

Reply:
after we merge #62 ..